homeassistant/components/onvif/parsers.py

```diff
@@ -353,16 +362,38 @@ async def async_parse_last_reset(uid: str, msg) -> Event:
 
     Topic: tns1:Monitoring/OperatingTime/LastReset
     """
+    date_time = datetime_or_zero(msg.Message._value_1.Data.SimpleItem[0].Value)
```

Comment:
The below being said, this should be inside the `try` block.

Reply:
Done